workflows/python/sdk/order-processor/workflow_activities.py

```diff
@@ -0,0 +1,101 @@
+            return OrderResult(processed=False)    
+    
+    yield ctx.call_activity(process_payment_activity, input=PaymentRequest(request_id=order_id, item_being_purchased=orderPayload.item_name, amount=orderPayload.total_cost, quantity=orderPayload.quantity))
+
+    try:
+        yield ctx.call_activity(update_inventory_activity, input=PaymentRequest(request_id=order_id, item_being_purchased=orderPayload.item_name, amount=orderPayload.total_cost, quantity=orderPayload.quantity))
+    except Exception:
+        yield ctx.call_activity(notify_activity, input=Notification(message=f'Order {order_id} Failed!'))
+        return OrderResult(processed=False)
+
+    yield ctx.call_activity(notify_activity, input=Notification(message=f'Order {order_id} has completed!'))
+    return OrderResult(processed=True)
+
+def notify_activity(ctx: WorkflowActivityContext, input: Notification):
+    # Create a logger
+    logger = logging.getLogger('NotifyActivity')
+    logger.info(input.message)
+
+
+def process_payment_activity(ctx: WorkflowActivityContext, input: PaymentRequest):
+    logger = logging.getLogger('ProcessPaymentActivity')
+    logger.info(f'Processing payment: {input.request_id} for {input.quantity} {input.item_being_purchased} at {input.amount} USD')
+    logger.info(f'Payment for request ID {input.request_id} processed successfully')
+
+
+def verify_inventory_activity(ctx: WorkflowActivityContext,
+                                input: InventoryRequest) -> InventoryResult:
+    logger = logging.getLogger('VerifyInventoryActivity')
+
+    logger.info(f'Verifying inventory for order {input.request_id} of {input.quantity} {input.item_name}')
+    inventoryItem: InventoryItem
+    with DaprClient(f'{address["host"]}:{address["port"]}') as client:
+        result = client.get_state(store_name, input.item_name)
+    if result.data is None:
+        return InventoryResult(False, None)
+    res_json=json.loads(str(result.data.decode('utf-8')))
+    logger.info(f'There are {res_json["quantity"]} {res_json["name"]} available for purchase')
+    inventoryItem = InventoryItem(item_name=input.item_name, per_item_cost=res_json['per_item_cost'], quantity=res_json['quantity'])
+
+    if res_json['quantity'] >= input.quantity:
+        return InventoryResult(True, inventoryItem)
+    return InventoryResult(False, None)
+
+def update_inventory_activity(ctx: WorkflowActivityContext,
+                                input: PaymentRequest) -> InventoryResult:
+    logger = logging.getLogger('UpdateInventoryActivity')
+
+    logger.info(f'Checking inventory for order {input.request_id} for {input.quantity} {input.item_being_purchased}')
+    with DaprClient(f'{address["host"]}:{address["port"]}') as client:
+        result = client.get_state(store_name, input.item_being_purchased)
+        res_json=json.loads(str(result.data.decode('utf-8')))
+        newQuantity = res_json['quantity'] - input.quantity
```

Comment:
The variable naming throughout this file is quite inconsistent. I suggest you have a consistent naming contention using `snake_case` instead of `camelCase`, since that's more common for Python.

Reply:
this is important to be Pythonic

Reply:
tried to make it consistent in file. resolving for now, please un-resolve and let me know if something is missed.